ArduPlane/system.cpp

```diff
@@ -149,7 +149,8 @@ void Plane::init_ardupilot()
 
     // setup frsky
 #if FRSKY_TELEM_ENABLED == ENABLED
-    frsky_telemetry.init(serial_manager);
+    // setup frsky, and pass a number of parameters to the library
+    frsky_telemetry.init(serial_manager, (uint8_t *)&control_mode);
```

Comment:
Good catch. Looks like there could be several ways to solve this but I would appreciate if there was one you could recommend...